packages/core/src/components/label/label.tsx

```diff
@@ -0,0 +1,40 @@
+import { Component, h, Prop } from '@stencil/core';
+import clsx from 'clsx';
+
+@Component({
+  tag: 'dso-label',
+  styleUrl: 'label.scss',
+  shadow: true
+})
+
+export class Label {
+  @Prop()
+  status?: 'primary' | 'info' | 'success' | 'warning' | 'danger';
+
+  private static statusMap = new Map<string, string>([
+    ['primary', 'Primair'],
+    ['info', 'Info'],
+    ['success', 'Succes'],
+    ['warning', 'Waarschuwing'],
+    ['danger', 'Gevaar']
+  ]);
+
+  @Prop()
+  button?: boolean;
+
+  render() {
+    return (
+      <span class={clsx('dso-label', { [`dso-label-${this.status}`]: this.status } )}>
+        <span class="sr-only">{this.status ? Label.statusMap.get(this.status) : `Label`}: </span>
+        {this.button && (
+          <button type="button">
```

Comment:
Button krijgt (voorlopig?) geen eigen web component, deze markup zal je moeten verplaatsen naar de aanroep van `<dso-label>` zodat hij in de `<slot></slot>` terecht komt.

Reply:
Nee, je moet de huidige nunjucks aanroep blijven gebruiken bij de aanroepen van `<dso-label>`